src/HTMLGenerator/converter.py

```diff
@@ -27,13 +28,16 @@ def convert(top_level_output_directory, top_level_input_directory, json_with_fil
     # Writing the top, preload, input and output path to the currentConfiguration.txt file.
     try:
         with open(current_configuration_absolute_path, 'w') as current_configuration_file:
-            current_configuration_file.write(
+            if "topPath" in json_with_filepaths:
+                current_configuration_file.write(
                 '/top:' + str(top_level_output_directory / json_with_filepaths["topPath"]) + "\n")
-            current_configuration_file.write(
+            if "preloadedFiles" in json_with_filepaths:
+                current_configuration_file.write(
                 '/preload:' + str(Path(json_with_filepaths["preloadedFiles"])) + "\n")
             current_configuration_file.write(
                 '/input:' + str(top_level_input_directory / json_with_filepaths["inputPath"]) + "\n")
```

Comment:
How come there's no `if` condition for "inputPath"?

Reply:
If a user want to convert files inputPath being empty does not make much sense

Reply:
Yes, but the code should still be robust enough to handle human errors like that. We all make mistakes, the tool should be capable of handling those. Which reminds me, we need error handling for this as well. Since we're saying "topPath" and "inputPath" are required inputs, we can print an error from this python script.